### PR TITLE
feat(extension-chrome): Optimize sign data UI

### LINKED ISSUE #99: [UI/UX] `wallet_signData` overflowed
## Reproduce

1. open this [link](https://github.com/ckb-js/nexus/pull/86/commits/3105ba3b5c4537c6f61d6ddaf7a69028543a9862)
2. run `ckb.request({method: "wallet_enable"})`
3. run `await ckb.request({method:'wallet_fullOwnership_signData', params: { data:'0x68656c6c6f20e4bda0e5a5bd20e38193e38293e381abe381a1e381af20f09f918b' }})`

image

### PR DESCRIPTION
**Pull request by IronLu233**
This PR optimize the UI of signing data page.

image
image
image

closes #99

### CODE REVIEW

**Comment by IronLu233** [comment]
https://github.com/ckb-js/nexus/blob/main/packages/extension-chrome/src/services/ownership/fullOwnership.ts#L136
```ts
// TODO how to get url?
const { password } = await platformService.requestSignData({ data: bytes.hexify(payload.data), url: '' });
```
Missing pass the requester URL.

**Comment by IronLu233** [comment]
image
image
image
Now each notification window works well.

**Comment by homura** [comment]
@IronLu233  can you please help to resolve conflicts

File: packages/extension-chrome/__tests__/rpc/wallet.test.ts [hunk 1]
```diff
@@ -97,6 +97,7 @@ describe('RPC wallet_fullOwnership', () => {
     expect(fullOwnershipService.signData).toBeCalledWith({
       data: '0x1234',
       lock: { codeHash: '0x1234', hashType: 'type', args: '0x01' },
+      url: 'internal://nexus-wallet.io',
```
Comment (homura) [inline packages/extension-chrome/__tests__/rpc/wallet.test.ts:100]: Replace with the constant `MOCK_PLATFORM_URL`

File: packages/extension-chrome/__tests__/rpc/wallet.test.ts [hunk 2]
```diff
@@ -107,7 +108,10 @@ describe('RPC wallet_fullOwnership', () => {
     jest.spyOn(fullOwnershipService, 'signTransaction').mockImplementation(() => Promise.resolve([]));
     await request('wallet_fullOwnership_signTransaction', { tx: createFakeTransaction() });
     expect(fullOwnershipService.signTransaction).toBeCalledTimes(1);
-    expect(fullOwnershipService.signTransaction).toBeCalledWith({ tx: createFakeTransaction() });
+    expect(fullOwnershipService.signTransaction).toBeCalledWith({
+      tx: createFakeTransaction(),
+      url: 'internal://nexus-wallet.io',
```
Comment (homura) [inline packages/extension-chrome/__tests__/rpc/wallet.test.ts:113]: Replace with the constant MOCK_PLATFORM_URL

File: packages/extension-chrome/src/services/ownership/fullOwnership.ts [hunk 3]
```diff
@@ -131,9 +130,12 @@ export function createFullOwnershipService({
 
       return signatures;
     },
-    signData: async (payload: SignDataPayload): Promise<Signature> => {
+    signData: async (payload) => {
       // TODO how to get url?
```
Comment (homura) [inline packages/extension-chrome/src/services/ownership/fullOwnership.ts:134]: we can rm it now

**Review by homura — APPROVED** [review]

File: docs/rpc.md [hunk 4]
```diff
@@ -616,10 +616,6 @@ GetPaginateItemsPayload.cursor
 
  `Optional` **change**: ``"external"`` \| ``"internal"``
 
-**`See`**
-
-[BIP44 Change](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#change)
-
```
Comment (homura) [inline docs/rpc.md:622]: Have you modified `packages/types/src/rpc/index.ts` ?

Comment (IronLu233) [reply docs/rpc.md:622]: Never. but why it become reverted?

Comment (homura) [reply docs/rpc.md]: I have no idea, but maybe we can merge the `main` and rebuild it again

Comment (IronLu233) [reply docs/rpc.md]: https://github.com/ckb-js/nexus/pull/108/files#diff-e002cf77525bdb6b3a89c3a0ae777fae997367025e035ec7da68ce53269005d0
I rebuild the package and commit it. It still have changes on auto generated doc.

**But they are only code formatting changes by prettier.**

Comment (homura) [reply docs/rpc.md]: I see, I'll append `rpc.md` to `.prettierignore`

**Pull request merged**